src/test/java/reposense/git/GitDiffTest.java

```diff
@@ -31,4 +31,11 @@ public void diffCommit_latestCommitHash_emptyResult() {
     public void diffCommit_nonexistentCommitHash_throwsRunTimeException() {
         GitDiff.diffCommit(config.getRepoRoot(), NONEXISTENT_COMMIT_HASH);
     }
+
+    @Test
+    public void diffCommit_commitContainingSubmodule_ignoresSubmodule() {
+        GitCheckout.checkout(config.getRepoRoot(), "850-GitDiffTest-commitWithSubmodule_success");
+        String diffResult = GitDiff.diffCommit(config.getRepoRoot(), LATEST_COMMIT_HASH);
```

Comment:
not so ideal to diff it with `LATEST_COMMIT_HASH` because it does not make sense in this context - the latest hash is not the latest.

Perhaps use the id of the empty tree `4b825dc642cb6eb9a060e54bf8d69288fbee4904` to get the diff from initial till to-date